lib/index.d.ts

```diff
@@ -0,0 +1,78 @@
+declare module "callable-instance" {
+  const CALL: unique symbol;
+  export type SCALL = typeof CALL;
```

Comment:
I think we should remove this type alias. It's not saving much and it adds an extra concept for people to be confused by. In this file we can use `typeof CALL` directly, and any consumer can use `typeof Callable.CALL`.